aws-rds-dbsubnetgroup/src/main/java/software/amazon/rds/dbsubnetgroup/Translator.java

```diff
@@ -13,12 +15,16 @@
 import software.amazon.awssdk.services.rds.model.ModifyDbSubnetGroupRequest;
 import software.amazon.awssdk.services.rds.model.Subnet;
 import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
 import software.amazon.rds.common.handler.Tagging;
 
 public class Translator {
+    static final String RDS = "rds";
+    static final String RESOURCE_PREFIX = "es:";
```

Comment:
Could you please elaborate this prefix? I'm not sure what es stands for

Reply:
Good catch. I made it by mistake while extracting constant from code before commit.